cmd/dorepl/main.go

```diff
@@ -60,10 +61,16 @@ func main() {
 		flag.PrintDefaults()
 		log.Fatalf("At this time, the REPL requires you to provide an API token")
 	}
-
-	gc := godo.NewClient(oauth2.NewClient(oauth2.NoContext,
+	var opts []godo.ClientOpt
+	if *apiURL != "" {
+		opts = append(opts, godo.SetBaseURL(*apiURL))
+	}
+	gc, err := godo.New(oauth2.NewClient(oauth2.NoContext,
```

Comment:
What's the diff between `godo.New` and `godo.NewClient`?

Reply:
@nanzhong `New` takes the func. opts